### PR TITLE
Testing PR webhooks

### PR DESCRIPTION
**Pull request by xitonix**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ A good example of such an environment is when we use infrastructure as service i
 
 This tool is a very simple wrapper around ssh to simplify loading identity files without being in the directory where you keep the SSH keys.
 
-# Installation
+## Installation
```
Comment (xitonix) [inline README.md:9]: Test

Comment (xitonix) [reply README.md:9]: one more

Comment (sandbox-alg) [reply README.md:9]: You

**Review by sandbox-alg — COMMENTED** [review]
Commented

**Review by sandbox-alg — CHANGES_REQUESTED** [review]
Change it

**Review by sandbox-alg — APPROVED** [review]
Approved

**Pull request merged**